src/tags/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames/bind'
+import Tag from '../tag'
+import styles from './index.css'
+import { getDataset } from '../utils'
+
+const cx = cn.bind(styles)
+
+const getTags = (tags = [], onDelete, readOnly, disabled, labelRemove) =>
+  tags.map(tag => {
+    const { _id, label, tagClassName, dataset } = tag
+    return (
+      <li className={cx('tag-item', tagClassName)} key={`tag-item-${_id}`} {...getDataset(dataset)}>
+        <Tag
+          label={label}
+          id={_id}
+          onDelete={onDelete}
+          readOnly={readOnly}
+          disabled={disabled}
+          labelRemove={labelRemove}
+        />
+      </li>
+    )
+  })
+
+class Tags extends PureComponent {
+  static propTypes = {
+    tags: PropTypes.array,
+    onTagRemove: PropTypes.func,
+    readOnly: PropTypes.bool,
+    disabled: PropTypes.bool,
+    texts: PropTypes.object,
+    children: PropTypes.node,
+  }
+
+  render() {
+    const { tags, onTagRemove, texts = {}, disabled, readOnly, children } = this.props
+    return (
+      <ul className={cx('tag-list')}>
+        {getTags(tags, onTagRemove, readOnly, disabled, texts.labelRemove)}
+        {children && <li className={cx('tag-item')}>{children}</li>}
+        {!children && (!tags || (tags && tags.length === 0)) && (
```

Comment:
> !tags || (tags && tags.length === 0)

Seems like a bit of excessive branching. Can we use ES6 defaults and eliminate branches or better yet, eliminate it altogether since it seems redundant. 

```jsx
{!children && (!tags || (tags && tags.length === 0)) && (
  <li className={cx('tag-item')}>
    <span className={cx('placeholder')}>{texts.placeholder || 'Choose...'}</span>
  </li>
)}
```

If no children, you'd render the placeholder regardless of tags or no tags, no?

I was wondering if we could do something like:

```jsx
const lastItem = children || <span className={cx('placeholder')}>{texts.placeholder || 'Choose...'}</span>
return (
  <ul className={cx('tag-list')}>
    {getTags(tags, onTagRemove, readOnly, disabled, texts.labelRemove)}
    <li className={cx('tag-item')}>{lastItem}</li>
  </ul>
)
```

and maybe even inline `getTags` since there's no good reason for it to exist outside (it was done earlier to reduce code complexity when it was inside `Input`'s `render` block).

Thoughts?